new-site/docs-site/content/0.19.0/guide/typesense-dynamodb.md

```diff
@@ -0,0 +1,240 @@
+# DynamoDB Integration with Typesense
+
+Hey there! This post give you a guide to integrate Typesense cluster with AWS DynamoDB by setting up a trigger with AWS Lambda.
+
+![Typesense DynamoDB Integration Chart](~@guide-images/typesense-dynamodb/typesense-dynamodb.svg)
+
+## Step 1: Create Typesense Cluster
+
+Sign up for a new account in [Typesense Cloud](https://cloud.typesense.org/) and get Endpoint URL, Port number and API key.
```

Comment:
After that line... 

We're using Typesense Cloud for this walk-through since we need a public Typesense endpoint for the Lambda function to be able write to. 

You can also self-host Typesense on a server of your choice. See this (link to Installing Typesense section) for more details on how to self-host Typesense.